content/docs/1.6.0/deploy/install/install-with-fleet.md

```diff
@@ -0,0 +1,162 @@
+
+    Ensure the fleet.yaml file includes the following configuration to install Longhorn in longhorn-system namespace:
+
+    ```yaml
+    defaultNamespace: longhorn-system
+    ```
+
+    Ensure the fleet.yaml file includes the following configuration to [ignore modified CRDs](https://fleet.rancher.io/bundle-diffs):
+
+    ```yaml
+    diff:
+      comparePatches:
+      - apiVersion: apiextensions.k8s.io/v1
+        kind: CustomResourceDefinition
+        name: engineimages.longhorn.io
+        operations:
+        - {"op": "replace", "path": "/status"}
+      - apiVersion: apiextensions.k8s.io/v1
+        kind: CustomResourceDefinition
+        name: nodes.longhorn.io
+        operations:
+        - {"op": "replace", "path": "/status"}
+      - apiVersion: apiextensions.k8s.io/v1
+        kind: CustomResourceDefinition
+        name: volumes.longhorn.io
+        operations:
+        - {"op": "replace", "path": "/status"}
+    ```
+
+    Finally, ensure the fleet.yaml file includes the following section to specify the version of Longhorn helm chart to be installed:
+
+    ```yaml
+    helm:
+      repo: https://charts.longhorn.io
+      chart: longhorn
+      version: v1.6.0 # Replace with the Longhorn version you'd like to install or upgrade to
+      releaseName: longhorn
+    ```
+
+    The complete fleet.yaml file would be like:
+
+    ```yaml
+    defaultNamespace: longhorn-system
+    helm:
+      repo: https://charts.longhorn.io
+      chart: longhorn
+      version: 1.6.0
+      releaseName: longhorn
+    diff:
+      comparePatches:
+      - apiVersion: apiextensions.k8s.io/v1
+        kind: CustomResourceDefinition
+        name: engineimages.longhorn.io
+        operations:
+        - {"op": "replace", "path": "/status"}
+      - apiVersion: apiextensions.k8s.io/v1
+        kind: CustomResourceDefinition
+        name: nodes.longhorn.io
+        operations:
+        - {"op": "replace", "path": "/status"}
+      - apiVersion: apiextensions.k8s.io/v1
+        kind: CustomResourceDefinition
+        name: volumes.longhorn.io
+        operations:
+        - {"op": "replace", "path": "/status"}
+    ```
+
+2. Create GitRepo CR points to your Gitops repository:
+
+    ```bash
+    cat > longhorn-gitrepo.yaml << "EOF"
+    apiVersion: fleet.cattle.io/v1alpha1
+    kind: GitRepo
+    metadata:
+      name: longhorn
+      namespace: fleet-local
+    spec:
+      repo: https://github.com/your-username/your-gitops-repo.git
+      revision: main
+      paths:
+      - .
+    EOF
+    ```
+
+3. Apply the GitRepo CR:
+
+    ```bash
+    kubectl apply -f longhorn-gitrepo.yaml
+    ```
+
+4. Check GitRepo Status:
+
+    Check the status of the GitRepo to ensure it has been created and synced successfully.
+
+    ```bash
+    kubectl -n fleet-local get fleet -w
+    ```
+
+    Once it's been synced successfully, Longhorn should be installed successfully, too:
```

Comment:
```suggestion
1. Verify that Longhorn was installed successfully.
```

Reply:
I made this a separate step so users know that there are two verification actions.